tools/terraform/main.tf

```diff
@@ -9,10 +9,10 @@ provider "aws" {
 terraform {
   // these values should match what is bootstrapped in ./remote-state
   backend "s3" {
-    bucket         = "fleet-terraform-remote-state"
+    bucket         = "${var.prefix}fleet-terraform-remote-state"
```

Comment:
Unfortunately variables cannot be used in the terraform block:

```
$ AWS_PROFILE=fleet terraform init -reconfigure
Initializing modules...

Initializing the backend...
╷
│ Error: Variables not allowed
│
│   on main.tf line 12, in terraform:
│   12:     bucket         = "${var.prefix}fleet-terraform-remote-state"
│
│ Variables may not be used here.
╵

╷
│ Error: Variables not allowed
│
│   on main.tf line 15, in terraform:
│   15:     dynamodb_table = "${var.prefix}fleet-terraform-state-lock"
│
│ Variables may not be used here.
╵
```

I think this is related to the line [in this section](https://www.terraform.io/language/settings/backends/configuration#using-a-backend-block):

> A backend block cannot refer to named values (like input variables, locals, or data source attributes).

Though it is possible to use other [means of configuration](https://www.terraform.io/language/settings/backends/configuration#partial-configuration) which I think would work in this scenario.